exercises/diamond/src/test/java/DiamondPrinterTest.java

```diff
@@ -0,0 +1,111 @@
+                                     "      T                                     T      ",
+                                     "     U                                       U     ",
+                                     "    V                                         V    ",
+                                     "   W                                           W   ",
+                                     "  X                                             X  ",
+                                     " Y                                               Y ",
+                                     "Z                                                 Z",
+                                     " Y                                               Y ",
+                                     "  X                                             X  ",
+                                     "   W                                           W   ",
+                                     "    V                                         V    ",
+                                     "     U                                       U     ",
+                                     "      T                                     T      ",
+                                     "       S                                   S       ",
+                                     "        R                                 R        ",
+                                     "         Q                               Q         ",
+                                     "          P                             P          ",
+                                     "           O                           O           ",
+                                     "            N                         N            ",
+                                     "             M                       M             ",
+                                     "              L                     L              ",
+                                     "               K                   K               ",
+                                     "                J                 J                ",
+                                     "                 I               I                 ",
+                                     "                  H             H                  ",
+                                     "                   G           G                   ",
+                                     "                    F         F                    ",
+                                     "                     E       E                     ",
+                                     "                      D     D                      ",
+                                     "                       C   C                       ",
+                                     "                        B B                        ",
+                                     "                         A                         ")));
```

Comment:
👍 for those two additional tests.  I'm in favor of opening a PR following on this to make those tests canonical-- should there be a 'discussion' issue first?

^^^ `DIamondPrinter.java` should probably be `DiamondPrinter.java`, no?